src/org/labkey/snd/SNDController.java

```diff
@@ -1006,7 +1006,7 @@ public ApiResponse execute(SimpleApiJsonForm form, BindException errors)
             if (!errors.hasErrors())
             {
                 String projectIdrev = json.getString("projectIdRev");
-                String note = json.getString("note");
+                String note = json.has("note") ? json.getString("note") : null;
```

Comment:
Do we need to also check for an empty string?

Reply:
json.has("note") && json.getString("note").length() > 0

Reply:
the empty string is being checked in our Mobile application while adding the value and purging it if not needed and the note element will not be present in the Json object and when it is not present , i am setting null. i validated with postman and also ran application with the code modified above.

Reply:
Works for me - thanks!